tests/testthat/sample_app_right_sidebar/program/fxn/plots.R

```diff
@@ -12,33 +12,29 @@ attr(mtcars, "show_rownames") <- TRUE
 # -- plotting functions
 
 plot2ggplot <- function() {
-    plot <- ggplot(data = mtcars, aes(x = wt, y = mpg)) +
+    ggplot(data = mtcars, aes(x = wt, y = mpg)) +
         geom_point(aes(color = cyl)) +
         theme(legend.justification = c(1, 1),
               legend.position = c(1, 1),
-              legend.title = element_blank()) +
+              legend.title    = element_blank()) +
```

Comment:
do we need to align with legend.justification

Reply:
done